catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sops-compliance-checker
+  description: |
+    Tool that looks for SOPS files in a repository
+    and makes sure that they are configured in the desired fashion
+  annotations:
+    backstage.io/techdocs-ref: dir:.
+  tags:
+    - go
+    - image
+    - kubernetes
+    - sops
```

Comment:
I think the `image` and `kubernetes` tags don't make sense here because it's not dealing with container images or anything Kubernetes-specific. Probably copy-pasted from somewhere by accident?

Reply:
Kubernetes I removed, but left image as we will be building and running this from an image